src/pykeen/datasets/base.py

```diff
@@ -281,12 +290,12 @@ def __init__(
             create_inverse_triples=create_inverse_triples,
         )
 
-    def _get_paths(self) -> Tuple[str, str, str]:  # noqa: D401
+    def _get_paths(self) -> Tuple[pathlib.Path, pathlib.Path, pathlib.Path]:  # noqa: D401
```

Comment:
what code relies on this? maybe it still expects them to come out as strings

Reply:
It is private method, and only used in this file, as far as I saw.